test/unit/api/api.test.js

```diff
@@ -153,14 +157,6 @@ describe('API', () => {
           expect(response.body.HttpError).toMatchSnapshot();
         });
     });
-
-    it('should return 500 if an error happens', () => supertest(app)
```

Comment:
Probot won't throw an error on a 500 like it did in 7x. I couldn't find a way to replicate this situation anymore.